### DISCUSSION TITLE
Adding full build plate to hardware folders

### DISCUSSION THREAD

**Discussion by doudar** [doudar/SmartSpin2k#263] [Ideas]
@eMadman , do you think it's beneficial (lower the barrier to entry for new builders) if we included a prusaslicer or cura file with all the required stl's for your mod prearranged on a build plate? Maybe there's a way to save a prearranged multipart .stl?

Let me know if this is something you'd be interested in doing, otherwise I might try it in a week or so (vacation for me next week - weeee!).

**Comment by eMadman** [comment]
I'll see if I can plate it as a 3mf or STL file. Fusion 360 isn't great for exporting files but I might have something that can get it arranged and plated.  I'll describe my recommended print settings in the readme.md as well.  Should probably update the Bom to be a bit more specific and link to some video instructions on how to use the heat set inserts while I'm in there.  I can probably get it done before the weekend is over!

Side note... My final revisions from last night seem to be pretty well perfect!  the command tape is a thing of the past.  It's held in place with zip ties and some rubber non skid stickers.  Blends right in!

@doudar woo vacation!  Couple more weeks till my Christmas downtime as well

**Reply by eMadman** [reply]
@doudar what are your thoughts on me plating it with just the outer files?  IE, the body, bike mount, slides/arms, and the bike mount.  My thinking is that the gears might be difficult to print and together, these are about 8 hours at 0.2 layers with my printer knocking out some very fast speeds speeds around 160mm/s.  I think the gears might be a difficult print for people with weaker first layer adhesion... Especially since I like the look of those parts with an accent color - also makes it easier to see how well I've applied grease on the gears.

Printing this version of the design now - if it turns out well and feels good after assembly, I'll push the changes and make another pull request.  I'll include the plated STL with it.

![image](https://user-images.githubusercontent.com/24726844/144723548-a4319fe2-5871-4eb9-9407-58aae224ca0c.png)

**Reply by doudar** [reply]
If you think it needs two plates, I’m good with that.

**Reply by tootechy** [reply]
@eMadman Did you create this plate anywhere? I'm looking for your case mods for the IC4 (Parts shown on the plate above), links gave me a 404 in the 'Case v2 Assembly discussion'. My printer died a few months ago, so I had to put this project on hold for a bit.  Just got a new printer, so diving back in to this project again!

**Reply by eMadman** [reply]
I haven't created and published a plate.  I had one for a while but pulled it eventually.  Main reason is because there are direct mount designs available for more bikes and I wanted users to have the flexibility to plate it themselves.

You'll want to print the following files from the direct mount mod folder:

* Both case files (left and right)
* the IC4 Bike mount
* the IC4 arm
* _arm hook.stl
* ic4 arm

And from the main project folders: 
* knob cup
* knob insert for the schwinn IC4
* 40t gear
* 11t gear

The screws, heat set inserts, and other components you'll need are listed out here in the bom here: https://github.com/doudar/SmartSpin2k/tree/develop/Hardware/MODS/Direct%20Mount%20Mod

Let me know how it turns out for you and if you run into any challenges along the way!

**Reply by tootechy** [reply]
Ah, under Direct Mount! I looked everywhere but there and I knew I saw it somewhere! I've got most of the components, and printing the gear assemblies now. 

Thanks!

**Reply by eMadman** [reply]
That would be my fault - it started off as the IC4 mount only... and then I managed to get some measurements and fitments going for other bikes.  The restructured folders should be easier to follow along with now.  I hope!
